client-app/shared/payment/components/payment-processing-redirection.vue

```diff
@@ -69,7 +64,7 @@ async function initPayment() {
 }
 
 function redirect() {
-  location.href = redirectUrl.value;
+  location.href = redirectUrl.value!;
```

Comment:
add check

Reply:
It will be called only if button is enabled, and it's disabled if !redirectUrl.value, so safe to assert